programs/leap-cli11/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+add_library(leap-cli11 INTERFACE)
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/include)
+target_include_directories(leap-cli11
```

Comment:
You probably just need `target_include_directories(leap-cli11 INTERFACE include)` here (current source dir is implied). not seeing off hand why you need `include_directories()` either

Reply:
Thank you, fixed, as you said, it works perfectly with just `target_include_directories(leap-cli11 INTERFACE include)`